be/src/storage/vectorized/schema_change.cpp

```diff
@@ -913,35 +912,35 @@ OLAPStatus SchemaChangeHandler::_do_process_alter_tablet_v2(const TAlterTabletRe
 
     std::shared_lock base_migration_rlock(base_tablet->get_migration_lock(), std::try_to_lock);
     if (!base_migration_rlock.owns_lock()) {
-        return OLAP_ERR_RWLOCK_ERROR;
+        return Status::InternalError("base tablet get migration r_lock failed");
     }
     std::shared_lock new_migration_rlock(new_tablet->get_migration_lock(), std::try_to_lock);
     if (!new_migration_rlock.owns_lock()) {
-        return OLAP_ERR_RWLOCK_ERROR;
+        return Status::InternalError("new tablet get migration r_lock failed");
     }
 
     if (base_tablet->keys_type() == KeysType::PRIMARY_KEYS) {
         Status st = new_tablet->updates()->load_from_base_tablet(request.alter_version, base_tablet.get());
         if (!st.ok()) {
             LOG(WARNING) << "schema change new tablet load snapshot error " << st.to_string();
-            return OLAP_ERR_OTHER_ERROR;
+            return st;
+            //return OLAP_ERR_OTHER_ERROR;
```

Comment:
not used code can be removed.

Reply:
done https://github.com/StarRocks/starrocks/pull/453/commits/6398dd310cefb667074015ae1adfeea29dcc165c